src/oas.md

```diff
@@ -3796,6 +3813,85 @@ animals:
 </aliens>
 ```
 
+###### XML With `null` Values
+
+Recall that the schema validates the in-memory data, not the XML document itself.
+The properties of the `"metadata"` element are omitted for brevity as it is here to show how the `null` value is represented.
+
+```json
```

Comment:
Did we remove the JSON examples from the specification? Or was that just for 3.2

Reply:
I answered my own question. The JSON is still there in 3.1